applications/launchpad/gui-react/src/containers/WalletContainer/WalletSetupBox.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTheme } from 'styled-components'
+
+import { useAppSelector } from '../../store/hooks'
+import SvgTariSignet from '../../styles/Icons/TariSignet'
+import WalletPasswordWizard from '../../containers/WalletPasswordWizard'
+import MiningBox from '../../containers/MiningContainer/MiningBox'
+import { MiningBoxStatus } from '../../containers/MiningContainer/MiningBox/types'
+import t from '../../locales'
+import {
+  selectTariContainers,
+  selectTariMiningState,
+} from '../../store/mining/selectors'
+
+const WalletSetupBox = () => {
+  const theme = useTheme()
+
+  const nodeState = useAppSelector(selectTariMiningState)
+  const containersState = useAppSelector(selectTariContainers)
+
+  return (
+    <MiningBox
```

Comment:
You can use plain `<NodeBox />` instead of `<MiningBox />`:

```
return (
    <NodeBox
      title='Tari wallet'
      tag={{
        content: 'Start here',
      }}
      style={{
        position: 'relative',
        boxShadow: theme.shadow40,
        borderColor: 'transparent',
      }}
    >
      <WalletPasswordWizard submitBtnText={t.wallet.setUpTariWalletSubmitBtn} />
    </NodeBox>
  )
```

Reply:
regarding help prompt - there's an issues for wallet prompts [here](https://github.com/Altalogy/tari/issues/20)

Reply:
- [x] used NodeBox
- [ ] added big tari icon